### DISCUSSION TITLE
Why is there no "fetch tables from database" button?

### DISCUSSION THREAD

**Discussion by cgarvie** [Budibase/budibase#3689] [Help]
I am trying to use a MySQL database from another server as the primary datastore for my Budibase project. I can successfully connect to the MySQL database from Budibase (confirmed by running queries and getting valid results), but there is no "fetch tables from database" button.

The documentation I am referring to is here in the "Fetching tables" section:

https://docs.budibase.com/building-apps/data/tables/postgresql-mysql-and-sql-server

What should I do after connecting successfully to the MySQL database? I need to fetch the tables, not run manual SQL queries

<img width="1440" alt="Screen Shot 2021-12-06 at 2 18 58 PM" src="https://user-images.githubusercontent.com/1124396/144804354-1b476a7c-56ef-43f4-9bd9-3ad6a8930579.png">

Thanks!

**Comment by shogunpurple** [comment]
Hi @cgarvie How are you running budibase? 

Are you using our cloud or are you self hosted? It's possible that if you are self-hosting, your budibase installation may be out of date. I can't seem to reproduce this in any of my environments.

**Reply by cgarvie** [reply]
It's a self-hosted install created 24 hours ago by using the DigitalOcean image.

**Reply by shogunpurple** [reply]
Thanks @cgarvie - it's possible that your installation is out of date. We've submitted an updated image to the digitalocean marketplace which should be available over the next few days. You can upgrade your installation from the "Update" tab in your budibase portal.

**Reply by cgarvie** [reply]
Thanks
